libs/spark/src/NavBarItem.tsx

```diff
@@ -0,0 +1,97 @@
+import React, { FC } from 'react';
+import { withStyles } from '@material-ui/core/styles';
+import {
+  Button,
+  ButtonBase,
+  ButtonBaseProps as MuiButtonBaseProps,
+} from '@material-ui/core';
+import { ButtonProps } from './Button';
+import { theme } from './styles';
+// import clsx from 'clsx';
+// import { capitalize } from './utils';
+
+export type NavBarItemProps = ButtonProps;
+
+export const NavBarItem = withStyles({
+  root: {
+    height: 48,
+    borderRadius: 8,
+    padding: 8,
+    marginLeft: 8,
+    cursor: 'pointer',
+    '&$disabled': {
+      opacity: '0.56',
+      pointerEvents: 'none',
+      cursor: 'default',
+    },
+    '&$hover': {
+      backgroundColor: theme.palette.grey.light,
+    },
+    '&$focused': {
+      border: theme.palette.blue[5],
+      outline: 'none',
+    },
+    '&$checked': {
+      color: theme.palette.blue[5],
+    },
+  },
+  label: {
+    color: theme.palette.text.onLightLowContrast,
+    fontSize: '1.125rem',
+  },
+  disabled: {},
+  focused: {},
+})(Button);
+
+// const SparkNavBarItem: FC<ButtonProps> = ({
```

Comment:
Not sure if you want this to be commented on, but since this component doesn't have >1 variant, we can avoid going along the same lines as Spark's current `Button` implementation. You can use Mui's `Button` directly. Note that you're already doing this based on what you're styling above.